### PR TITLE
Feat/pci ai notebooks header notebook

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-ai-notebooks/src/components/labels/LabelsForm.component.tsx [hunk 1]
```diff
@@ -105,47 +85,51 @@ const LabelsForm = React.forwardRef<HTMLInputElement, LabelsFormProps>(
...
+                </li>
+              ))}
+            </ul>
+            <p data-testid="configured-labels">
+              {t('numberOfConfiguredLabels', {
+                count: configuredLabels.length,
+                max: CONFIGURATION_CONFIG.maxLabelNumber,
+                context: `${configuredLabels.length}`,
+              })}
+            </p>
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/components/labels/LabelsForm.component.tsx:130]: yep you are right. Fixed.

File: packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/label/useEditLabel.hook.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export function useEditLabel({ onError, onSuccess }: MutateLabelProps) {
+  const mutation = useMutation({
+    mutationFn: (labelInfo: EditLabelProps) => {
+      return editLabel(labelInfo);
+    },
+    onError,
+    onSuccess,
+  });
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/notebook/label/useEditLabel.hook.tsx:20]: Yes but it should invalidate the notebook as labels are displayed using the notebook object. 
Fixed.

File: packages/manager/apps/pci-ai-notebooks/src/lib/durationHelper.ts [hunk 3]
```diff
@@ -75,13 +75,13 @@ export function convertSecondsToTimeString(seconds: number) {
...
-    timeStringParts.push(`${days}d`);
+    timeStringParts.push(short? `${days}d` :  `${days} jours`);
   }
   if (hours > 0) {
-    timeStringParts.push(`${hours}h`);
+    timeStringParts.push(short ? `${hours}h` : `${hours} heures`);
   }
   if (minutes > 0) {
-    timeStringParts.push(`${minutes}m`);
+    timeStringParts.push(short ?  `${minutes}m` : `${minutes} minutes`);
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/lib/durationHelper.ts:84]: yes, fixed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/Notebook.layout.tsx [hunk 4]
```diff
@@ -55,6 +57,7 @@ export default function NotebookLayout() {
   if (!notebook) {
     return (
       <>
+        <NotebookHeader.Skeleton />
         <TabsMenu.Skeleton />
         Loading your notebook data
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/Notebook.layout.tsx:62]: Fixed

File: packages/manager/apps/pci-ai-notebooks/src/components/labels/LabelsForm.component.tsx [hunk 5]
```diff
@@ -15,53 +13,35 @@ import {
...
       form.reset();
     };
 
     const removeLabel = (indexToRemove: number) => {
-      const newLabels = labelValue.filter(
+      const newLabels = configuredLabels.filter(
         (_, index) => index !== indexToRemove,
       );
-      onChange(newLabels);
+      onRemove(newLabels);
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/components/labels/LabelsForm.component.tsx:44]: fixed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/_components/NotebookHeader.component.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+            size="badge"
+            disabled={!isRunningNotebook(notebook.status.state)}
+          >
+            <A
+              href={notebook.status.url}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <div className="flex flex-row gap-1 items-center text-white capitalize">
+                Ouvrir {notebook.spec.env.editorId}
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/_components/NotebookHeader.component.tsx:82]: fixed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Labels.component.tsx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+            .filter(
+              ([labkey]) =>
+                labkey !== OVH_TAGS_CONFIG.id &&
+                labkey !== OVH_TAGS_CONFIG.type,
+            )
+            .map(([labKey, value]) => (
+              <Badge className="py-1" key={labKey} variant={'info'}>
+                <div className="flex flex-row gap-1">
+                  <span key={`span_${labKey}`}>
+                    {labKey} = {value}
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Labels.component.tsx:81]: yep, fixed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/LifeCycle.component.tsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+              <TableRow
+                key={`${state.date}-${state.state}`}
+                className="text-sm"
+              >
+                <TableCell>
+                  <Check className="size-4 text-sky-600" />
+                </TableCell>
+                <TableCell className="font-semibold">{state.state}</TableCell>
+                <TableCell className="font-semibold">
+                  {format(state.date, 'yyyy-MM-dd HH:mm:ss')}
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/LifeCycle.component.tsx:32]: Fixed

File: packages/manager/apps/pci-ai-notebooks/src/components/ui/slider.tsx [hunk 9]
```diff
@@ -18,8 +18,7 @@ const Slider = React.forwardRef<
     <SliderPrimitive.Track className="relative h-2 w-full grow overflow-hidden rounded-full bg-gray-100">
       <SliderPrimitive.Range className="absolute h-full bg-primary" />
     </SliderPrimitive.Track>
-    <SliderPrimitive.Thumb className="block h-5 w-5 rounded-full border-2 border-primary bg-background ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50" />
-  </SliderPrimitive.Root>
+ </SliderPrimitive.Root>
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/components/ui/slider.tsx]: FIxed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Labels.component.tsx [hunk 10]
```diff
@@ -0,0 +1,79 @@
+import { useTranslation } from 'react-i18next';
+import { useNotebookData } from '../../Notebook.context';;
```
[the remark this answers is not in the document]
Comment (jperchoc) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Labels.component.tsx:2]: I think you also need to change the order of the imports

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Resources.component.tsx [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+
+const Resources = () => {
+  const { notebook } = useNotebookData();
+  const { t } = useTranslation('pci-ai-notebooks/notebooks/notebook/dashboard');
+  return (
+    <div data-testid="resources-container">
+      {notebook.spec.resources.gpu > 0 ? (
+        <div className="flex flex-col">
+          <div className="flex flex-row gap-2 items-center">
+            <h5>Power</h5>
```
Comment (jperchoc) [inline packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Resources.component.tsx:20]: Do we display "Power" for every language?

File: packages/manager/apps/pci-ai-notebooks/src/components/labels/LabelsForm.component.tsx [hunk 12]
```diff
@@ -15,53 +13,50 @@ import {
...
+import { CONFIGURATION_CONFIG } from '../order/configuration/configuration.constants';
+import { useLabelForm } from '@/components/labels/useLabelForm.hook';
+import { Badge } from '../ui/badge';
 
 interface LabelsFormProps {
-  labelValue: ai.Label[];
-  onChange: (newLabels: ai.Label[]) => void;
+  configuredLabels: ai.Label[];
+  onAdd?: (newLabel: ai.Label) => void;
+  onDelete?: (newLabel: ai.Label) => void;
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/components/labels/LabelsForm.component.tsx]: fixed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Labels.component.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Labels.component.tsx]: Fixed

File: packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Resources.component.tsx (hunk 11 above)
Comment (abullet33) [reply packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Resources.component.tsx]: Fixed

**Review by jperchoc — APPROVED** [review]

**Pull request merged**